### PR TITLE
Put loose files in folders

### PR DESCRIPTION
**Pull request by tskarhed**
If all of these have the same filestructure, it is much easier to target with the lilypond CLI. This will then make generating stuff from 11ty super easy; just the listing the numbered SVGs and MIDIs might be a bit more difficult. Now everything is on the format `SongName/main.ly`.

### CODE REVIEW

File: folders_for_all.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    if [ -f "$f" ]
+    then
+        filenameAndExtension=$(basename -- "$f")
+        filename="${filenameAndExtension%.*}"
+        echo $f
+        echo $filename
+        mkdir ./songs/$filename
+        mv $f ./songs/$filename/main.ly
+    fi
+done
```
Comment (MrTengil) [inline folders_for_all.sh:15]: Newline
`
`

**Review by MrTengil — CHANGES_REQUESTED** [review]
4got newline....

Is the lilypond backwards compatible?

**Comment by Kongzhu79** [comment]
Lilypond is not completely backwards compatible. It is possible to translate between versions but I have not been able to do it myself.

I can change the file structure, but I need some time to do it. Have trouble finding it at the moment...

File: folders_for_all.sh (hunk 1 above)
Comment (tskarhed) [reply folders_for_all.sh:15]: Here, have a newline:
```


```

**Comment by tskarhed** [comment]
It throws an error when building, but the output looks good. :shrug: I guess it may cause some issues when we build in CI.

**Pull request closed without merging**